src/main/java/com/alibaba/fastjson/parser/deserializer/Jdk8DateCodec.java

```diff
@@ -67,7 +67,12 @@
                     formatter = DateTimeFormatter.ofPattern(format);
                 }
             }
-
+            if (text == null) {
```

Comment:
@Test
    public void testJson() {


        String json = "{\"ld\":\"\"}";

        Vo vo = JSON.parseObject(json, Vo.class);
        System.out.println(vo);

    }

    public static class Vo{
        private LocalDate ld;

        public LocalDate getLd() {
            return ld;
        }

        public void setLd(LocalDate ld) {
            this.ld = ld;
        }
    }


com.alibaba.fastjson.JSONException: Text '' could not be parsed at index 0